src/lib/components/Logo.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  let clazz = '';
+  export { clazz as class };
+  const finalClass = `colors max-h-20 ${clazz}`;
+  //read setting from .env
+  const useLogoImg = import.meta.env.VITE_LOGO;
```

Comment:
Maybe add the branding/theme specific env variables to the example one.

Reply:
I will tackle this in another ticket